common/kvstore/KeyValueStore.cc

```diff
@@ -258,21 +262,15 @@ void OwnedKeyValueStore::clear() {
 
 string_view OwnedKeyValueStore::readString(string_view key) const {
     auto rawData = read(key);
-    if (!rawData) {
+    if (rawData.data == nullptr) {
         return string_view();
     }
-    size_t sz;
-    memcpy(&sz, rawData, sizeof(sz));
-    string_view result(((const char *)rawData) + sizeof(sz), sz);
+    string_view result((const char *)rawData.data, rawData.len);
```

Comment:
iiuc, the point of this PR is to delete these three `memcpy`, and the only reason why we were using `memcpy` before was to populate `sz`?

Can you remind me why we expect the `rawData.data` to live longer than `result`?

Reply:
>iiuc, the point of this PR is to delete these three memcpy, and the only reason why we were using memcpy before was to populate sz?

The primary point of this PR is that I need to know the length of binary AST data stored in kvstore for my work on compressed ASTs. After changing the api of `read`, I realized I could simplify existing code and did so!

> Can you remind me why we expect the rawData.data to live longer than result?

The lifetime of `rawData.data` is tied to the database. When the database is closed, it's unmapped from memory and the memory is reclaimed.

Note that I'm not changing the lifetime of this `string_view` at all in this PR, I'm just removing size from the stored data.